### PR TITLE
Add env vars required by elastic_integraion plugin.

### PR DESCRIPTION
**Pull request by mashhurs**
### Description

Adds `ELASTICSEARCH_TREEISH` env var which will be used by `elastic_integration` plugin.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -28,7 +28,10 @@ RUN cp /usr/share/logstash/logstash-core/versions-gem-copy.yml /usr/share/logsta
 # NOTE: since 8.0 JDK is bundled as part of the LS distribution under $LS_HOME/jdk
 ENV PATH="${PATH}:/usr/share/logstash/vendor/jruby/bin:/usr/share/logstash/jdk/bin"
 ENV LOGSTASH_SOURCE="1"
+ENV LOGSTASH_PATH="/usr/share/logstash"
```
Comment (robbavey) [inline Dockerfile:31]: It looks like the elastic integration filter has the same usage of [LOGSTASH_PATH](https://github.com/elastic/logstash-filter-elastic_integration/blob/main/Gemfile#L5) as regular plugins. Is there a specific reason why the integration filter requires this? Does this have implications for other users of `.ci`?

Comment (mashhurs) [reply Dockerfile:31]: [gradle task](https://github.com/elastic/logstash-filter-elastic_integration/blob/main/build.gradle#L149) requires it.

Comment (mashhurs) [reply Dockerfile:31]: In the gradle property we have a definition but it is `LOGSTASH_PATH=../../ls` 🤦

Comment (mashhurs) [inline Dockerfile:31]: Let's [update `gradle.properties`](https://github.com/elastic/logstash-filter-elastic_integration/blob/main/gradle.properties#L1)
```suggestion
```

Comment (mashhurs) [reply Dockerfile]: Removed from here and updated the gradle.properties: https://github.com/elastic/logstash-filter-elastic_integration/pull/194/commits/be7a98cca0f2b7312fce36a010d54797c0fa40b1

**Review by robbavey — APPROVED** [review]
LGTM

**Pull request merged**